R/data-gather-clean.R

```diff
@@ -65,16 +65,17 @@ make_clean_table <- function(data, section_lookup_table,
     "label",
     "response"
   )]
+  data <- change_logical_responses(data)
   data
 }
 
 #' Clean experiment variables
+#' to having one known `variable`, ID50, that ends in a number,
 #'
-#' Multiple experiments lead to `variable`s with names of the
 #' form "age_range1", "age_range2". This will remove the number at
+#' Multiple experiments lead to `variable`s with names of the
 #' the end of `variable` and append to the `step` name.
 #' Note that this only happens if the number is > 0. This is due
-#' to having one known `variable`, ID50, that ends in a number,
 #' but is not related to the multiple experiment issue.
 #'
```

Comment:
I think some of these lines got rearranged

Reply:
Haha, oh my. Thank you for catching that. I fixed that. It seems like that was the only section that got rearranged when I was going through the conflicts. Some of that was actually outdated since we changed how the numbers were found. I updated it and re-ran the documentation tool. I also noticed I had missed the stringr import in the description. Added that.